app/src/main/java/com/babylon/wallet/android/presentation/account/history/HistoryViewModel.kt

```diff
@@ -398,11 +398,16 @@ data class State(
             else -> null
         }
 
-    val fungibleResources: List<Resource.FungibleResource>
-        get() = accountWithAssets?.assets?.knownFungibles.orEmpty()
-
-    val nonFungibleResources: List<Resource.NonFungibleResource>
-        get() = accountWithAssets?.assets?.knownNonFungibles.orEmpty()
+    // we use this to show  for now we filter out LSU -
+    val fungibleResourcesUsedInFilters: List<Resource.FungibleResource>
+        get() = accountWithAssets?.assets?.tokens?.map {
+            it.resource
+        }.orEmpty() + accountWithAssets?.assets?.poolUnits?.map {
+            it.stake
+        }.orEmpty()
+
+    val nonFungibleResourcesUsedInFilters: List<Resource.NonFungibleResource>
+        get() = accountWithAssets?.assets?.nonFungibles?.map { it.collection }.orEmpty()
```

Comment:
Neither for stake claims here I presume?

Reply:
yes